src/ValueObject/Route.php

```diff
@@ -25,6 +25,10 @@ public function getParameters(): ?array
 
     public function addParameters(array $parameters): void
```

Comment:
typehint moet dan toch ook `?array` zijn?

Reply:
Die methode returnt niets. Is een setter op dit object.

Reply:
En je zou nooit addParameters callen met null. Je kan ze wel weglaten bij de `new Route`, als er nog geen zijn op dat moment, maar addX doen met niets lijkt mij nogal nutteloos.